src/app/shared/shared.module.ts

```diff
@@ -1,15 +1,30 @@
-import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
 import { RouterModule } from '@angular/router';
-
+import { MaterialModule } from '../material.module';
+import { DynamicFormGroupComponent } from './dynamic-form/dynamic-form-group.component';
+import { DynamicFormComponent } from './dynamic-form/dynamic-form.component';
 import { LayoutComponent } from './layout/layout.component';
-import { NotFoundComponent } from './not-found/not-found.component';
 import { LocaleSelectComponent } from './locale-select/locale-select.component';
-import { MaterialModule } from '../material.module';
+import { NotFoundComponent } from './not-found/not-found.component';
```

Comment:
If you have some order-imports setting in your editor or some extension please disable it.

Reply:
Disabled it, I had.